connectors/sources/jira.py

```diff
@@ -364,21 +381,24 @@ async def _get_projects(self):
         Yields:
             project: Project document to get indexed
         """
-        async for response in self._api_call(url_name=PING):
-            timezone = await response.json()
-            self.timezone = timezone["timeZone"]
-
-        async for response in self._api_call(url_name=PROJECT):
-            response = await response.json()
-            for project in response:
-                yield {
-                    "_id": f"{project['name']}-{project['id']}",
-                    "_timestamp": iso_utc(
-                        when=datetime.now(pytz.timezone(self.timezone))
-                    ),
-                    "Type": "Project",
-                    "Project": project,
-                }
+        try:
+            async for response in self.jira_client.api_call(url_name=PING):
```

Comment:
That's because we get a `timezone` from the endpoint `/myself`(PING) which is also used for pinging the Jira and that is later on used to make `_timestamp` consistent of all Jira objects.

Reply:
Maybe you could wrap that functionality in a method called `_get_timezone` and then set it. Then it's more obvious at first sight, what's happening